extensions.json

```
@@ -1566,6 +1566,10 @@
       "version": "11.4.0",
       "checkout": "v11.4.0"
     },
+    {
+      "id": "tootone.org-mode",
+      "repository": "https://github.com/vscode-org-mode/vscode-org-mode"
```

- The `publisher` in the extension's [package.json](https://github.com/vscode-org-mode/vscode-org-mode/blob/e8cc69868af427f5fd03c8ad125b493cbf2e8292/package.json#L7) seems to be `vscode-org-mode`, not `tootone`
- The latest official version is `1.0.0`. I think we don't want to publish `1.1.0-SNAPSHOT` (or whatever state the default branch currently is in), so it's better to publish `1.0.0` specifically, by checking out the relevant tag, branch, or commit

```suggestion
      "id": "vscode-org-mode.org-mode",
      "repository": "https://github.com/vscode-org-mode/vscode-org-mode",
      "version": "1.0.0",
      "checkout": "e8cc69868af427f5fd03c8ad125b493cbf2e8292"
```